.github/workflows/maven.yml

```diff
@@ -0,0 +1,75 @@
+name: Arquillian-Cube Maven Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'doc/**'
+      - '.circleci/**'
+env:
+  # failsafe.groups configuration depends on the env setup.
+  FAILSAFE_GROUPS: ''
+  DOCKER_HOST: 'unix:///var/run/docker.sock'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
I am not sure we need to create a platform matrix as this will require linux containers, so this won't ever work on windows and probably neither on macos.

Reply:
i.e. I would just do `runs-on: ubuntu-latest` and move on

Reply:
got it !

Reply:
I guess `matrix` aims at supporting more than just one JDK, as it was in the initial PR: https://github.com/arquillian/arquillian-cube/pull/1274/files#diff-5dbf1a803ecc13ff945a08ed3eb09149a83615e83f15320550af8e3a91976446R19

But just my take.
A question, though: wouldn't be better to have them working on linux and file a separate tracker for expanding? Short living (hopefully) and easy to review PRs would be the reason.
WDYT?

Reply:
JDK matrix for sure - but os matrix not really useful.